### PR TITLE
SPCTR-3403: Fix - Container background video opacity issue

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Container background video overlay opacity is not working as expected in frontend, because of the value updating incorrectly.
https://brainstormforce.atlassian.net/browse/SPCTR-3403 
### Screenshots

https://d.pr/v/FF8lEu

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]
Look good to me

File: includes/blocks/container/frontend.css.php [hunk 1]
```diff
@@ -126,7 +126,7 @@
 
 // Handle backward opacity for video.
 // If this was saved in the updated version, backgroundVideoOpacity will be 0, and this will be skipped.
-if ( 'video' === $attr['backgroundType'] && ! empty( $attr['backgroundVideoOpacity'] ) ) {
+if ( 'video' === $attr['backgroundType'] && ! empty( $attr['backgroundVideoOpacity'] ) && 0 !== $attr['backgroundVideoOpacity'] ) {
```
Comment (adi3890) [inline includes/blocks/container/frontend.css.php:129]: @rajmohanreddy774 why are we adding this extra logic
if $a in not empty if will not be equal to 0.

Can you please describe it a bit.

**Review by adi3890 — APPROVED** [review]

File: includes/blocks/container/frontend.css.php (hunk 1 above)
Comment (rajmohanreddy774) [reply includes/blocks/container/frontend.css.php:129]: @adi3890 We are making use of backgroundVideoOpacity once and making it 0, for backward case, so this condition helps us update the attribute by comparing.

Comment (rajmohanreddy774) [reply includes/blocks/container/frontend.css.php]: @adi3890 Removal of `0 !== $attr['backgroundVideoOpacity'] ` doesn't effect the functionality,
! empty check is sufficient in this case,
Hence removed `0 !== $attr['backgroundVideoOpacity'] `.

**Review by bsf-zanev — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**